int32 id
string frame
bool keep_transform
---

